src/main/blackbox/blackbox.c

```diff
@@ -1078,7 +1078,7 @@ static void loadMainState(timeUs_t currentTimeUs)
 
     // log the currentPidSetpoint values applied to the PID controller
     for (int i = 0; i < XYZ_AXIS_COUNT; i++) {
-        blackboxCurrent->setpoint[i] = lrintf(pidGetPreviousSetpoint(i));
+        blackboxCurrent->setpoint[i] = lrintf(pidGetPreviousSetpoint(i)*10.0f);
```

Comment:
How much blackbox log file increment will these bring?

Reply:
Good question. It is now for discussion and to help tuning some issues. I need to do some tests for comparison.

Reply:
@DusKing1 It is now opt-in to use new scaling for some values and it is meant only to support better debugging. Default OFF. Make sense?